### PR TITLE
Modification to tf.keras.activations.tanh().

### PR DESCRIPTION
**Pull request by aenjon**
learing

### CODE REVIEW

**Comment by rthadur** [comment]
@aenjon please sign CLA

**Review by alextp — APPROVED** [review]

**Comment by rthadur** [comment]
@aenjon please rebase your branch

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by rthadur** [comment]
Closing this PR as similar changes have been pushed here #26603
